ci/script/predicate_functions.sh

```diff
@@ -1,67 +1,21 @@
 function is_mri {
-  if ruby -e "exit(!defined?(RUBY_ENGINE) || RUBY_ENGINE == 'ruby')"; then
```

Comment:
Can we revert all these changes for now, it makes diffing harder and its easy to clean up post release.